src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +31,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Products DATA was successfully fetched from DB. Products:{}{}",
```

Comment:
The log message should be more informative. It should include the user ID to indicate whose cart's products were fetched.